tests/e2e_tests/test_find.py

```diff
@@ -24,17 +18,17 @@ def test_find_download(domains, authenticator_fund_config, page: Page):
     request_data_page.filter_organisations("A District Council From Hogwarts")
     request_data_page.filter_outcomes("Economy", "Transport")
     request_data_page.filter_returns_period(from_quarter=2, from_year="2022/2023", to_quarter=3, to_year="2022/2023")
-
     request_data_page.select_file_format("XLSX (Microsoft Excel)")
 
-    request_data_page.request_data()
+    find_request_data_success_page: FindRequestDataSuccessPage = request_data_page.request_data()
```

Comment:
No. We don't. Any half decent code editor will know the type. 
I chose to be explicit about it, because the methods that return a different instance, to act as if you get redirected, are not very obvious form their name that they return a different instance of a different type.

I'd like to leave them, if we want to keep this "redirect returns new instance approach".

Reply:
I disagree, but I won't push the point 👀